tests/Tests/500__jupyterhub/culler.robot

```diff
@@ -0,0 +1,169 @@
+*** Settings ***
+Documentation    Tests for the NB culler
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library          ../../../libs/Helpers.py
+Library          OpenShiftLibrary
+Suite Teardown   Teardown
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${DEFAULT_CULLER_TIMEOUT} =    31536000
+${CUSTOM_CULLER_TIMEOUT} =     600
+
+
+*** Test Cases ***
+Verify Default Culler Timeout
+    [Documentation]    Checks default culler timeout
+    [Tags]    Sanity    Tier1
+    ...       ODS-1255
+    Disable Culler
```

Comment:
Suite teardown?

Reply:
What do you refer to? `Disable Culler`? If so, it's already done in the Teardown keyword

Reply:
ops, sorry I meant to say Suite Setup, but it is another comment related to https://github.com/red-hat-data-services/ods-ci/pull/295/files/2e86723bff531d72835c322d254fb157dab751e5#r856147372.